modules/mod_ginger_collection/templates/page.collection_object.tpl

```diff
@@ -25,13 +25,15 @@
             </div>
         </div>
         <article class="main-content">
-            {% block title %}
+            
+            {% block page_title %}
                 <h1 class="page-title">{% include "beeldenzoeker/title.tpl" title=record['dcterms:title']|default:record.title %}</h1>
             {% endblock %}
-
+            
             {% if record.title[2] %}
                 <h2>{{ record.title[2] }}</h2>
             {% endif %}
+            
```

Comment:
Please don’t add empty lines like these: enable Editorconfig support in your editor.